pkg/csi_driver/controller.go

```diff
@@ -246,13 +244,10 @@ func (s *controllerServer) CreateVolume(ctx context.Context, req *csi.CreateVolu
 		}
 		if createErr != nil {
 			glog.Errorf("Create volume for volume Id %s failed: %v", volumeID, createErr)
-			if file.IsUserError(createErr) {
-				return nil, status.Error(codes.InvalidArgument, createErr.Error())
-			} else if file.IsTooManyRequestError(createErr) {
-				return nil, status.Error(codes.ResourceExhausted, createErr.Error())
-			} else {
-				return nil, status.Error(codes.Internal, createErr.Error())
+			if errCode := file.IsUserError(err); errCode != nil {
```

Comment:
This should have been createErr not err. I copy and pasted it and didn't catch this.